src/index.html

```diff
@@ -9,5 +9,51 @@
   </head>
   <body>
     <h1>Stars</h1>
+
+    <div class="stars stars--0">
+      <div class = "stars__star"></div>
+      <div class = "stars__star"></div>
+      <div class = "stars__star"></div>
+      <div class = "stars__star"></div>
+      <div class = "stars__star"></div>
+    </div>
```

Comment:
delete spaces in attributes
```suggestion
    <div class="stars stars--0">
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
      <div class="stars__star"></div>
    </div>
```